internal/handlers/not-found.go

```
@@ -33,9 +33,9 @@ import (
 // 	)
 // }
 
-func HandleNotFound(logger logger.Logger) http.Handler {
+func HandleNotFound() http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		w.Write([]byte("Wrong Path Bro\n"))
```

golangci-lint (1.62.2)

Error return value of `w.Write` is not checked

(errcheck)